components/eamxx/src/physics/rrtmgp/atmosphere_radiation.cpp

```diff
@@ -845,7 +846,12 @@ void RRTMGPRadiation::run_impl (const int dt) {
     const int kbot = nlay+1;
 
     // Compute diffuse flux as difference between total and direct; use YAKL parallel_for here because these are YAKL objects
-    parallel_for(Bounds<3>(nswbands,nlay+1,ncol), YAKL_LAMBDA(int ibnd, int ilev, int icol) {
+    Kokkos::parallel_for(Kokkos::RangePolicy<ExeSpace>(0,nswbands*(nlay+1)*ncol),
+                         KOKKOS_LAMBDA (const int idx) {
+      // CAREFUL: these are YAKL arrays, with "LayoutLeft". So make the indices stride accordingly
+      const int ibnd = (idx / ncol) / (nlay+1);
+      const int ilev = (idx / ncol) % (nlay+1);
+      const int icol =  idx % ncol;
       sw_bnd_flux_dif(icol,ilev,ibnd) = sw_bnd_flux_dn(icol,ilev,ibnd) - sw_bnd_flux_dir(icol,ilev,ibnd);
```

Comment:
Since these are YAKL arrays, I think they use 1-based instead of 0-based indexing, so should the indices here be `(icol+1,ilev+1,ibnd+1)`? I think the Kokkos parallel_for is looping over idx [0, nswbands*(nlay+1)*ncol - 1], so, e.g., for idx = 0, this gives icol = 0, but I think sw_bnd_flux_dif indexing starts at 1.